themes/src/main/package-lock.json

```diff
@@ -0,0 +1,1131 @@
+{
```

Comment:
Again, non-blocking but I believe this package-lock shouldn't be here. If anywhere, should be in `themes/src/main/resources/theme/keycloak/common/resources` where the `package.json` lives.